.github/spelling/known_words_corpus.txt

```diff
@@ -720,4 +720,49 @@ aws
 guidesmdx
 agentsicon
 ledgericon
-asiicon
\ No newline at end of file
+asiicon
+ganet
+googleapis
+toolresult
+xvfb
```

Comment:
This isn't a word 😅

Reply:
That is in a code snippet

Reply:
In `pages/guides/quickstart-with/Anthropic/computer-use-demo.mdx`

Reply:
does the code spell checker fail for spelling in code blocks?

Reply:
Nope,. it checks both content and block as far as i remember correctly but provides separate views in terminal